.travis.yml

```diff
@@ -52,6 +52,7 @@ before_install:
   - git clone https://github.com/informatics-isi-edu/ermrestjs.git
   - cd ermrestjs
   - sudo npm install
+  - bower install
```

Comment:
@karlcz  Nope, I added this line to install **ermrestjs**  **bower** dependencies when deploying CHAISE on TRAVIS. 

This is a part of Ermrestjs **PR** [#136](https://github.com/informatics-isi-edu/ermrestjs/pull/136) and has no implications on existing CHAISE code.